### ISSUE TITLE
Issues communicating with content server during deploy

### ISSUE THREAD

**Issue by RichardPoes** [Azure/static-web-apps#900]
Hi,
During a regular deploy, we suddenly got the following error.
![image](https://user-images.githubusercontent.com/30719799/186886263-e39ff228-ee10-4ca7-9e0d-92c9cd8070bb.png)
It seems that this issue is bigger than our repository, as other colleagues of me appear to have the same issue.
I cannot find any issues on https://status.azure.com/en-gb/status. Is anyone having the same issues, thereby confirming to me this problem is caused by Azure? Any outlook on when this will be resolved?

**Comment by lapin7771n** [comment]
+1

**Comment by kayijyc** [comment]
+1

**Comment by elisendamonserrat** [comment]
I am also having the same issue. I have been trying to run the job several times this morning but I keep getting the same issue

**Comment by JacobToftgaardRasmussen** [comment]
+1

**Comment by dexianta** [comment]
+1

**Comment by maickelr** [comment]
Experiencing the same problem here!

**Comment by hartman** [comment]
I tried to deploy a new static web app and the deploy failed with:  BadRequest
```
{
    "status": "Failed",
    "error": {
        "code": "InternalServerError",
        "message": "There was an unexpected InternalServerError.  Please try again later.  x-ms-correlation-request-id: b5bbca88-f200-46eb-a559-427b9149e0c0"
    }
}
```

**Comment by RichardPoes** [comment]
It is working again now.

**Comment by maickelr** [comment]
Hit and miss it seems... every now and then it works if you keep trying. But not consistently. 
Also getting 'InternalServerError now'

**Comment by RichardPoes** [comment]
You are correct. This seems to be happening to my colleagues as well. I find it rather strange that no other information is given. Both on Azure Status and on the [Twitter Account of Azure SWA](https://twitter.com/azurestaticapps) nothing is stated. I wonder where I would need to go for more info.

**Comment by mruben0** [comment]
Same for me

**Comment by bioniqMK** [comment]
+ 1

**Comment by Sinnaj94** [comment]
+ 1

**Comment by kayijyc** [comment]
It seems to work now. Been triggering the deploy intermittently for the past few hours 😅

**Comment by mishapos** [comment]
Hi, could you please share your deployment ID from earlier in the action logs and your default hostname. We can then check what was going on. Thanks!

**Comment by derekvantonder** [comment]
I am getting this error today consistently, I've tried a number of SWA builds and nothing works :(


Run Azure/static-web-apps-deploy@v1
  with:
    azure_static_web_apps_api_token: ***
    repo_token: ***
    action: upload
    app_location: /
    output_location: dist
/usr/bin/docker run --name a0dc7301bcc44303a4d8a2652cdea97d_40e64e --label 290506 --workdir /github/workspace --rm -e "INPUT_AZURE_STATIC_WEB_APPS_API_TOKEN" -e "INPUT_REPO_TOKEN" -e "INPUT_ACTION" -e "INPUT_APP_LOCATION" -e "INPUT_API_LOCATION" -e "INPUT_OUTPUT_LOCATION" -e "INPUT_API_BUILD_COMMAND" -e "INPUT_APP_ARTIFACT_LOCATION" -e "INPUT_APP_BUILD_COMMAND" -e "INPUT_ROUTES_LOCATION" -e "INPUT_SKIP_APP_BUILD" -e "INPUT_CONFIG_FILE_LOCATION" -e "INPUT_SKIP_API_BUILD" -e "INPUT_PRODUCTION_BRANCH" -e "INPUT_DEPLOYMENT_ENVIRONMENT" -e "INPUT_IS_STATIC_EXPORT" -e "HOME" -e "GITHUB_JOB" -e "GITHUB_REF" -e "GITHUB_SHA" -e "GITHUB_REPOSITORY" -e "GITHUB_REPOSITORY_OWNER" -e "GITHUB_RUN_ID" -e "GITHUB_RUN_NUMBER" -e "GITHUB_RETENTION_DAYS" -e "GITHUB_RUN_ATTEMPT" -e "GITHUB_ACTOR" -e "GITHUB_TRIGGERING_ACTOR" -e "GITHUB_WORKFLOW" -e "GITHUB_HEAD_REF" -e "GITHUB_BASE_REF" -e "GITHUB_EVENT_NAME" -e "GITHUB_SERVER_URL" -e "GITHUB_API_URL" -e "GITHUB_GRAPHQL_URL" -e "GITHUB_REF_NAME" -e "GITHUB_REF_PROTECTED" -e "GITHUB_REF_TYPE" -e "GITHUB_WORKSPACE" -e "GITHUB_ACTION" -e "GITHUB_EVENT_PATH" -e "GITHUB_ACTION_REPOSITORY" -e "GITHUB_ACTION_REF" -e "GITHUB_PATH" -e "GITHUB_ENV" -e "GITHUB_STEP_SUMMARY" -e "GITHUB_STATE" -e "GITHUB_OUTPUT" -e "RUNNER_OS" -e "RUNNER_ARCH" -e "RUNNER_NAME" -e "RUNNER_TOOL_CACHE" -e "RUNNER_TEMP" -e "RUNNER_WORKSPACE" -e "ACTIONS_RUNTIME_URL" -e "ACTIONS_RUNTIME_TOKEN" -e "ACTIONS_CACHE_URL" -e GITHUB_ACTIONS=true -e CI=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/_temp/_runner_file_commands":"/github/file_commands" -v "/home/runner/work/MWPressWebXXXX_XXX/MWPressWebXXXX_XXX":"/github/workspace" 290XXX:a0dc7301bcc44303a4d8a2652cdXXXXX
DeploymentId: 0ffb9247-0df7-XXXX-XXXX[19](https://github.com/derekXXXXXX/MWPressWebXXXX_XXX/actions/runs/3779129803/jobs/6424155493#step:4:19)-842a144[6](https://github.com/derekXXXXXX/MWPressWebXXXX_XXX/actions/runs/3779129803/jobs/6424155493#step:4:6)23[7](https://github.com/derekXXXXXX/MWPressWebXXXX_XXX/actions/runs/3779129803/jobs/6424155493#step:4:7)2

App Directory Location: '/' was found.
Looking for event info
Failed to contact content server, stopping...
An unknown exception has occurred

For further information, please visit the Azure Static Web Apps documentation at https://docs.microsoft.com/en-us/azure/static-web-apps/
If you believe this behavior is unexpected, please raise a GitHub issue at https://github.com/azure/static-web-apps/issues/
Exiting

**Help!**

<img width="1107" alt="image" src="https://user-images.githubusercontent.com/31227484/209502059-5f3e3cf3-3744-4055-b722-d1de374c9255.png">

**Comment by derekvantonder** [comment]
Deployment ID is: **DeploymentId: 0ffb9247-0df7-434a-ac19-842a14462372**

**Comment by mishapos** [comment]
@derekvantonder This is likely related to #1023 for which the issue has been resolved. Are your deployments succeeding now?

**Comment by dvantonder-vieb** [comment]
Yes, looks like it to me. My deployments are working again as of 28 Dec.

**Comment by cmydur** [comment]
Facing same issues here for my deployment , DeploymentId: 9b7f6cf4-3e71-48ef-8ada-fcb2021f6b49

**Comment by clarkin-dev** [comment]
Facing this issue for DeploymentId: 4c291308-f507-4552-9e36-8c0198884be0
